### ISSUE TITLE
Incorrect vocab URI

### ISSUE THREAD

**Issue by acka47** [skohub-io/skohub-docker-vocabs#4]
Submitted by @julsraemy on 2022-04-12 via email:

> For our research project, we are using SkoHub to deploy controlled 
> vocabularies in SKOS (at https://vocab.participatory-archives.ch/, the GitHub repository is 
> located here: https://github.com/Participatory-Image-Archives/skohub-docker-vocabs), 
> we have noticed that the url is not correct and the "name of the 
> vocabulary" is repeated twice:
> 
> https://vocab.participatory-archives.ch/vocab.participatory-archives.ch/brunner/id03d.de.html  instead 
> of https://vocab.participatory-archives.ch/brunner/id03d
> 
> 
> 
> Do we need to change something in the main.yml file 
> (https://github.com/Participatory-Image-Archives/skohub-docker-vocabs/blob/master/.github/workflows/main.yml)?

**Comment by acka47** [comment]
tl;dr SkoHub Vocabs currently only supports a setup where the vocab and concept URIs redirect to the SkoHub Vocabs URIs (e.g. using purl.org or w3id.org).

Here is what SkoHub Vocabs Docker version does:

Take the URIs of the vocabs and concepts, strip the protocol prefix ('`https://`) and attach the result to the URL where the SkoHub Vocabs site is publish. I.e. when I have the scheme URI `https://example.org/scheme` and publish the vocab at `https://example.org/skohub` it will result in `https://example.org/skohub/example.org/scheme` 

In other words, it is currently not possible to publish the vocab at the namespace that is indicated in the Turtle file.

As most people will use perma IDs like purl.org or w3id.org for their vocabs anyway, it is no problem to set up the right redirect so that the vocab _is_ served when resolving its URI. See https://github.com/skohub-io/swib20-workshop/blob/main/resources/publish-vocab.md#step-6-set-up-redirect-for-persistent-identifiers for how to do this.

(For documentation, this issue is a bot related to https://github.com/skohub-io/skohub-vocabs/issues/26).

@julsraemy, please let me know if I can close this issue.

**Comment by julsraemy** [comment]
Thank you for your answer @acka47. You can close the issue, we will most probably use ARKs as our persistent identifiers.
